README.Rmd

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Table of Contents"
-output: 
+output:
```

Comment:
I didn't intend to strip trailing whitespace in this file, but I figure it can't hurt so I left it in.